packages/client/wallets/smart-wallet/src/SmartWalletSDK.test.ts

```diff
@@ -1,5 +1,5 @@
 import { SmartWalletSDK } from "./SmartWalletSDK";
-import { SmartWalletSDKInitParams } from "./types/Config";
```

Comment:
Initially I just changed the case to `types/config`, but that messed up the build, so rather than dive into what was going on there I just renamed the file XD.